Завершаем вёрстку всех компонентов

InnokentyKedrov opened the pull request:
Добавлены: модальное окно, поповер, тултип, рэнж-слайдер, чекбоксы. Заданы состояния.

---
:mortar_board: [Завершаем вёрстку всех компонентов](https://up.htmlacademy.ru/htmlcss/33/user/1874843/tasks/12)

InnokentyKedrov committed:
Завершаем вёрстку всех компонентов

Добавлены: модальное окно, поповер, тултип, рэнж-слайдер, чекбоксы. Заданы состояния.

InnokentyKedrov committed:
Завершаем вёрстку всех компонентов

Добавил фавиконку, прошёл stylelint`ом по стилям, изменил вёрстку по валидности

InnokentyKedrov committed:
fix: label color

Сделал, чтобы подпись под инпутом загоралась красным, заменил кнопки на ссылки в товарах в index.html, удалил стрелку сортировки вниз, сделал якорные ссылки в index.html

InnokentyKedrov committed:
fix: focus on search

InnokentyKedrov committed:
fix: modal quantity

brewscriptor commented on catalog.html:
```diff
@@ -111,63 +147,101 @@ <h2 class="inner-header-title">Моноподы для селфи</h2>
+                <legend class="catalog-filter-title catalog-filter-title-price">Стоимость</legend>
+
+                  <div class="catalog-filter-range">
+                    <div class="catalog-filter-range-bar" style="left: 0px; width: 117px">
```
инлайн стили надо убрать в класс. если в текущий класс catalog-filter-range-bar не получается, то создать отдельный и положить туда

brewscriptor replied:
нужно везде проверить чтобы такого не было

brewscriptor commented on index.html:
```diff
@@ -1,46 +1,81 @@
-          <button class="search-button" type="submit">Найти</button>
-        </form>
+    <h1 class="visually-hidden">Магазин современных гаджетов Device</h1>
+    <a class="header-logo" href="#">
```
смотри, если у элемента есть href, то браузер будет фокусироваться тут, а нам этого не надо

brewscriptor commented on index.html:
```diff
@@ -61,373 +97,425 @@ <h1 class="visually-hidden">Магазин современных гаджето
+        <div class="footer-social-container">
+          <ul class="footer-social-list">
+            <li class="footer-social-item">
+              <a class="footer-social-link" href="https://www.facebook.com/htmlacademy">
```
ссылкам на сторонние ресурсы лучше добавлять target="_blank" чтобы пользователь открывать требуемую страницу в отдельной вкладке и оставался на нашем сайте

InnokentyKedrov committed:
fix: favicon

Заменил фавиконку, изменил разметку для label и input, добавил префиксы, удалил браузерные стрелки при input type="number"
